networkmanager/networkmanager.go

```diff
@@ -375,17 +375,31 @@ func (manager *NetworkManager) RemoveInstanceFromNetwork(instanceID, networkID s
 		return nil
 	}
 
+	var err error
+
 	if manager.trafficMonitoring != nil {
-		if err := manager.trafficMonitoring.stopInstanceTrafficMonitor(instanceID); err != nil {
-			return aoserrors.Wrap(err)
+		if errStopMonitoring := manager.trafficMonitoring.stopInstanceTrafficMonitor(instanceID); errStopMonitoring != nil {
+			log.Errorf("Can't stop monitoring instance traffic: %s", errStopMonitoring)
+
+			err = errStopMonitoring
```

Comment:
should error be wrapped for all these cases?
Looks like the common patter is to return the very first error.
Override of the return error is done under smth like; if errRem != nil && err == nil

Reply:
I used another pattern, because in case `if errRem != nil && err == nil` I can't print log error